indexer/blockchainetl/jobs/exporters/converters/decimal_to_int_item_converter.py

```diff
@@ -0,0 +1,12 @@
+from decimal import Decimal
+
+from blockchainetl.jobs.exporters.converters.simple_item_converter import SimpleItemConverter
+
+
+class DecimalToIntItemConverter(SimpleItemConverter):
```

Comment:
for?

Reply:
because insert method of clickhouse python sdk can not insert Decimal to UInt column in schema.
* [Block data](https://github.com/Octan-Labs/pipeline/blob/main/indexer/ethereumetl/mappers/block_mapper.py#L83)
* [Insert method](https://github.com/ClickHouse/clickhouse-connect/blob/main/clickhouse_connect/driver/client.py#L512-L539)

Reply:
why doesn't this effect exporter of export_all?